src/components/Sidebar.tsx

```diff
@@ -72,6 +80,7 @@ export default function Sidebar(props: Props) {
                 email: userDetails.email,
             });
             setData(LS_KEYS.SUBSCRIPTION, userDetails.subscription);
+            setData(LS_KEYS.FAMILY_DATA, userDetails.familyData);
```

Comment:
we should check if `familyData` is not undefined, and then only set it in localDB